### PR TITLE
User Details

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Logout User
- Change password
- Update Profile Details
- Get suggestions of user
- Follow/Unfollow User
- Check profiles of user

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Good work with RTK querry
* Looks Good

File: src/components/Buttons/FollowButton.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  const handleFollow = () => {
+    followUser(user._id);
+  };
+
+  const handleUnFollow = () => {
+    unFollowUser(user._id);
+  };
+
+  useEffect(() => {
```
Comment (animeshgarai09) [inline src/components/Buttons/FollowButton.jsx:48]: No use of useEffect here. Values from hook trigger re-render

**Review by animeshgarai09 — APPROVED** [review]

**Review by KanekarSnehal — APPROVED** [review]
No issues found, you can merge it

File: src/components/Buttons/FollowButton.jsx (hunk 1 above)
Comment (SukanyaSen530) [reply src/components/Buttons/FollowButton.jsx:48]: Thanks for pointing out, fixed -  [958c4dd](https://github.com/SukanyaSen530/Vibes-frontend/pull/6/commits/958c4dd5f4236ffdd50a5b347a77dc5a1b5a8260)

**Pull request merged**